packages/components/src/components/SignIn.vue

```diff
@@ -1,62 +1,192 @@
+        <input type="email" placeholder="email address" v-model="email" required />
+        <a
+          href="/"
+          class="btn btn-primary"
+          data-cy="email-activation-button"
+          @click.prevent="activateWithEmail"
+        >
+          Activate AppMap
+        </a>
+        <div class="accept-tos">
+          <p>
+            By activating AppMap, I agree to the
+            <a
+              class="link"
+              target="_blank"
+              href="https://appmap.io/community/terms-and-conditions.html"
+            >
+              AppMap Terms of Service<ExternalLinkIcon />
+            </a>
+          </p>
+        </div>
       </div>
 
       <div class="your-data">
-        <shieldIcon />
-        <p data-cy="your-data-text">
-          Authentication with GitHub or GitLab is used solely for issuing a license. AppMap runs in
-          your code editor, so your AppMaps and your source code stay on your machine.
-          <a href="https://appmap.io/security">Security FAQ<VExternalLinkIcon /></a>
+        <ShieldIcon />
+        <div data-cy="your-data-text" class="your-data-text">
+          <p>
+            Your email address is used solely to issue you an AppMap license. Your source code and
+            AppMaps stay on your machine when you use AppMap diagrams and runtime analysis. Consult
+            the
+            <a
+              class="link"
+              target="_blank"
+              href="https://appmap.io/community/terms-and-conditions.html"
+            >
+              Terms of Service<ExternalLinkIcon />
+            </a>
+            for data privacy terms of Navie AI.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div v-else class="finish-activation">
+      <div class="sidebar-title">
+        <h1 data-cy="title">Complete Activation</h1>
+        <p>
+          Check your inbox! We've sent you an email with a 6-digit verification code. Enter the code
+          below.
         </p>
       </div>
+
+      <div class="signin-buttons">
+        <p v-if="error" class="error">{{ error }}</p>
+        <input
+          v-model="verificationCode"
+          type="text"
+          class="temp"
+          placeholder="verification code"
+        />
+        <a
+          href="/"
+          class="btn btn-primary"
+          data-cy="verify-button"
+          @click.prevent="completeActivation"
+        >
+          Complete Activation
+        </a>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
 import AppMapLogo from '@/assets/appmap-full-logo.svg';
-import shieldIcon from '@/assets/shield-icon.svg';
-import VExternalLinkIcon from '@/assets/external-link.svg';
+import ShieldIcon from '@/assets/shield-icon.svg';
+import GitHubLogo from '@/assets/github-logo.svg';
+import GitLabLogo from '@/assets/gitlab-logo.svg';
+import ExternalLinkIcon from '@/assets/external-link.svg';
+
+const APPMAP_SERVER_URL = 'https://getappmap.com';
```

Comment:
This should ideally be a prop, considering it's a setting in VS Code

Reply:
Good call, I changed it :+1: